hypervm/httpdocs/htmllib/lib/lib.php

```diff
@@ -3845,6 +3845,18 @@ function is_centossix()
 	}
 }
 
+function is_centosseven()
```

Comment:
Hmm, I think, not now, but we have to think about RHEL7 too soon. I am not pretty sure if the virt-install.php is able as is now, and with those change, to be able to install Xen extension at RHEL7 (not CentOS7)

Reply:
Well virt-install.php as it is now is not able to install anything at rhel currently though its not that hard to make it happen using the same function check as for is_centos[version](), perhaps is_rhelseven() it can be implemented